src/GitHub.VisualStudio/UI/Views/PullRequestDetailView.xaml

```diff
@@ -400,11 +400,21 @@
                     
                     <te:SectionControl.Content>
                         <Grid>
+                            <Grid.Resources>
+                                <!-- We have to do contortions to get to the PullRequestDetailViewModel which contains the OpenFile and DiffFile
+                                     commands: the control which hosts the context menu must have the PullRequestDetailViewModel as its tag,
+                                     which is then set as the tag on the ContextMenu which is finally reacable from the MenuItem. -->
+                                <ContextMenu x:Key="FileContextMenu" Tag="{Binding PlacementTarget.Tag, RelativeSource={RelativeSource Mode=Self}}">
```

Comment:
Thanks for the advice @tocsoft - I've tried this out but I couldn't come up with a way of doing it that I was happy with: part of the problem is that you need the UserControl `DataContext` for the command and the changed file `DataContext` to pass into that command, so by definition you need two data contexts, which makes things difficult. I'd be interested in seeing your solution though if you wanted to give it a try!